playbooks/common/openshift-cluster/config.yml

```diff
@@ -17,6 +17,24 @@
       - package_version
       - docker_image_availability
       - docker_storage
+  when: ansible_distribution != "Fedora"
+
+- name: Verify Requirements - Fedora
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - docker_image_availability
+      - docker_storage
+  when: ansible_distribution == "Fedora"
```

Comment:
The check itself only runs on hosts in either `masters`, `nodes` or `etcd` groups, unless you specify otherwise like here we're using a `when` clause in Ansible. There is nothing distro-specific.

With that configuration in my previous comment, the check should ensure there is at least 0.1 GB free in `/var`, 1 GB in `/usr/local/bin` and 1 GB in `/tmp` (or equivalent temp dir).

Maybe requiring 1 GB for `/var` would even be fair, to avoid surprises related to no disk space left?